status/queues.py

```diff
@@ -101,100 +101,119 @@ class SequencingQueuesDataHandler(SafeHandler):
+                                    'and artifactid={} group by artifactid')
+                    #The final loading conc is defined in the Define Run format stap whose stepid is 1659
+                    final_lconc_query = ('select udfname, udfvalue from artifact_udf_view where udfname in (\'Final Loading Concentration (pM)\') '
+                                         'and artifactid in (select st.artifactid from stagetransition st, artifact_sample_map asm, sample, project '
+                                         'where st.artifactid = asm.artifactid AND sample.processid = asm.processid and sample.projectid = project.projectid '
+                                         'and project.projectid = {pnum} and generatedbyid in (select st.completedbyid '
+                                         'from stagetransition st, stage s, artifact_sample_map asm, sample, project '
+                                         'where st.stageid = s.stageid and s.stepid=1659 and st.artifactid = asm.artifactid and sample.processid = asm.processid '
+                                         'and sample.projectid = project.projectid AND project.projectid = {pnum} group by st.completedbyid) group by st.artifactid)')
+                    conc_query = ('select udfname, udfvalue from artifact_udf_view where udfname in (\'Concentration\') '
+                                        'and artifactid={}')
+
+                    #rerun
+                    is_rerun = False
+                    cursor.execute(rerun_query.format(record[0]))
+                    rerun_res = cursor.fetchone()[0]
+                    if rerun_res > 1:
+                        is_rerun = True
+
+                    #Final loading conc
+                    cursor.execute(final_lconc_query.format(pnum=record[5]))
+                    final_lconc_res = dict(cursor.fetchall())
+                    flc_novaseq = final_lconc_res.get('Final Loading Concentration (pM)')
+                    if flc_novaseq is not None:
+                        final_loading_conc = flc_novaseq
+
+                    #qPCR conc
+                    qpcr_conc_query = ('select art.artifactid, art.name from artifact_sample_map asm, artifact art '
+                                  'where processid=(select processid from artifact_sample_map where artifactid={} limit 1)'
+                                  'and art.artifactid=asm.artifactid and art.name=\'qPCR Measurement\';')
+                    cursor.execute(qpcr_conc_query.format(record[0]))
+                    c_res = cursor.fetchone()
+                    if c_res is None:
+                        conc_qpcr = 0.0
+                    else:
+                        cursor.execute(conc_query.format(c_res[0]))
+                        conc_qpcr = cursor.fetchone()[1]
+
+                pools[method][project]['final_loading_conc'] = final_loading_conc
+                pools[method][project]['plates'][container]['conc_pool_qpcr'] = conc_qpcr
```

Comment:
Please keep 2 digits for conc_qpcr

Reply:
The qPCR conc values are rounded in the LIMS frontend, this will show the value exactly as it is.